nuget/engine/nunit.engine.nuspec

```diff
@@ -34,37 +34,52 @@
   <files>
     <file src="LICENSE.txt" />
     <file src="NOTICES.txt" />
+    
     <file src="bin/net20/nunit.engine.dll" target="lib/net20" />
     <file src="bin/net20/nunit.engine.core.dll" target="lib/net20" />
     <file src="bin/net20/nunit.engine.api.dll" target="lib/net20" />
     <file src="bin/net20/testcentric.engine.metadata.dll" target="lib/net20" />
+    
     <file src="bin/agents/net20/nunit-agent.exe" target="contentFiles/any/agents/net20" />
     <file src="bin/agents/net20/nunit-agent.exe.config" target="contentFiles/any/agents/net20" />
     <file src="bin/agents/net20/nunit-agent-x86.exe" target="contentFiles/any/agents/net20" />
     <file src="bin/agents/net20/nunit-agent-x86.exe.config" target="contentFiles/any/agents/net20" />
     <file src="bin/agents/net20/nunit.engine.core.dll" target="contentFiles/any/agents/net20" />
     <file src="bin/agents/net20/nunit.engine.api.dll" target="contentFiles/any/agents/net20" />
     <file src="bin/agents/net20/testcentric.engine.metadata.dll" target="contentFiles/any/agents/net20" />
+    
     <file src="bin/agents/net40/nunit-agent.exe" target="contentFiles/any/agents/net40" />
     <file src="bin/agents/net40/nunit-agent.exe.config" target="contentFiles/any/agents/net40" />
     <file src="bin/agents/net40/nunit-agent-x86.exe" target="contentFiles/any/agents/net40" />
     <file src="bin/agents/net40/nunit-agent-x86.exe.config" target="contentFiles/any/agents/net40" />
     <file src="bin/agents/net40/nunit.engine.core.dll" target="contentFiles/any/agents/net40" />
     <file src="bin/agents/net40/nunit.engine.api.dll" target="contentFiles/any/agents/net40" />
     <file src="bin/agents/net40/testcentric.engine.metadata.dll" target="contentFiles/any/agents/net40" />
+    
+    <file src="bin/agents/netcoreapp3.1/nunit-agent.dll" target="contentFiles/any/agents/netcoreapp3.1" />
+    <file src="bin/agents/netcoreapp3.1/nunit-agent.dll.config" target="contentFiles/any/agents/netcoreapp3.1" />
+    <file src="bin/agents/netcoreapp3.1/nunit-agent.runtimeconfig.json" target="tools/agents/netcoreapp3.1" />
+    <file src="bin/agents/netcoreapp3.1/nunit.engine.core.dll" target="contentFiles/any/agents/netcoreapp3.1" />
+    <file src="bin/agents/netcoreapp3.1/nunit.engine.api.dll" target="contentFiles/any/agents/netcoreapp3.1" />
+    <file src="bin/agents/netcoreapp3.1/testcentric.engine.metadata.dll" target="contentFiles/any/agents/netcoreapp3.1" />
+    
```

Comment:
Not silly, I hope, because I originally thought so as well. However, I have read that the proper approach is to use the x86 version of the dotnet cli, so I removed those builds.

Reply:
Ah, cool! So how does this actually work for x86 assemblies then, sorry? I can see the code just calls `dotnet` based on the PATH var - does dotnet itself launch an x86 process if required?

Reply:
That's what I was referring to re TestAgency. It may need to add something to the command-line.

[Actually, I should have said AgentProcess rather than TestAgency. Thinking about two different engines is confusing. :confused: ]